moveit_experimental/moveit_jog_arm/src/jog_calcs.cpp

```diff
@@ -232,21 +231,31 @@ void JogCalcs::run(const ros::TimerEvent& timer_event)
     if (have_nonzero_twist_stamped_)
     {
       if (!cartesianJogCalcs(twist_stamped_, *joint_trajectory))
+      {
+        resetLowPassFilters(original_joint_state_);
```

Comment:
One of these calls to resetLowPassFilters is causing some really weird behavior.

Reply:
I believe this is fixed with the newest commits